### ISSUE TITLE
BGV_country_db_lookup example fails locating dataset

### ISSUE THREAD

**Issue by rosielickorish** [IBM/fhe-toolkit-linux#40]
**Describe the bug**
The BGV_country_db_lookup example is failing to locate the `countries_dataset.csv` and failing with the following error:

``` bash
what():  Error: This example failed trying to open the data file: ./examples/BGV_country_db_lookup/countries_dataset.csv
              Please check this file exists and try again.
```

**To Reproduce**
1. Start the toolkit hit `Build`
2. Launch the BGV_country_db_lookup demo using the `Launch` button
3. See the demo start to run but then fail trying to open the data file `countries_dataset.csv`

**Expected behavior**
The demo should run successfully after the toolkit is open and the Build, then Launch buttons are hit.

**Environment (please complete the following information):**
 - Host Info (The machine hosting the FHE Toolkit Docker container), please provide the output of running the following commands in a termina window:
   - `uname -` Darwin Kernel Version 19.5.0: Tue May 26 20:41:44 PDT 2020; root:xnu-6153.121.2~2/RELEASE_X86_64 x86_64
   - `ulimit -a`
```
-t: cpu time (seconds)              unlimited
-f: file size (blocks)              unlimited
-d: data seg size (kbytes)          unlimited
-s: stack size (kbytes)             8192
-c: core file size (blocks)         0
-v: address space (kbytes)          unlimited
-l: locked-in-memory size (kbytes)  unlimited
-u: processes                       2784
-n: file descriptors                10496
```
 - Docker version (run `docker -v` in Terminal): Docker version 19.03.12, build 48a66213fe
 - Toolkit platform (Fedora, CentOS, Ubuntu): All

**Additional context**
ms-vscode.cmake-tools version v1.4.2

**Comment by pparamde** [comment]
I'm hitting the same error with HPVS toolkit..

**Comment by rosielickorish** [comment]
Fix merged #41
